### DISCUSSION TITLE
Root view

### DISCUSSION THREAD

**Discussion by Ramilito** [Ramilito/kubectl.nvim#209] [Ideas]
The root view started as a hub to reach all the views and get an overview of the cluster.
Right now we have two great ways to reach views, the 1-5 numbers for frequent views and the alias search.

I don't see me using the root view for reaching views anymore, so what should the root view do?

1. Do we want to have it as a overview monitoring view that displays nodes up/down, pods running/pending/failed?
2. Remove it?

Other suggestions?

**Comment by Ramilito** [comment]
Would love your views (pun intended 😄) and ideas on this @mosheavni, @idelice

**Reply by mosheavni** [reply]
I like 1 better, make something new, fresh.
- header with cluster details (like in any view)
- namespaces with number of pods next to each namespace
clicking on a namespace will move us to pods view on that specific namespace
- `number of ready nodes / number of nodes`
- number of deployments maybe?
- some `kubectl top node` view to see node resource usage
- print number of failing pods (pods not on `Running` state and `containers_status ~= ready_containers .. "/" .. #containers`)
maybe a mapping to go to `~/.kubeconfig` or `$KUBECONFIG`

just threw a bunch of ideas, any of them stick?

**Reply by Ramilito** [reply]
All of them sound great!
The main take away is that you are voting for the monitor overview view 👍
